BenchmarkComponent/BenchmarkComponent.cpp

```diff
@@ -61,4 +65,63 @@ namespace winrt::BenchmarkComponent::implementation
     void ClassWithMultipleInterfaces::DefaultDoubleProperty(double val)
     {
     }
+
+    ClassWithMarshalingRoutines::ClassWithMarshalingRoutines()
+    {
+    }
+
+    hstring ClassWithMarshalingRoutines::DefaultStringProperty()
+    {
+        return hstring();
+    }
+
+    void ClassWithMarshalingRoutines::DefaultStringProperty(hstring val)
+    {
+    }
+    
+    Windows::Foundation::IInspectable ClassWithMarshalingRoutines::DefaultKeyValuePairProperty()
+    {
+        Windows::Foundation::Collections::ValueSet valueSet;
+        valueSet.Insert(hstring(L"key"), PropertyValue::CreateString(L"value"));
+        return valueSet.First().Current();
+    }
+
+    void ClassWithMarshalingRoutines::DefaultKeyValuePairProperty(Windows::Foundation::IInspectable val)
+    {
+    }
+
+    Windows::Foundation::IInspectable ClassWithMarshalingRoutines::DefaultNullableProperty()

```

Comment:
What do we mean by nullable property here?

Reply:
The returned value, which is an IReference projects to Nullable in cs. So I called it Nullable property.